Entities/AHuman.cpp

```diff
@@ -3355,6 +3345,16 @@ void AHuman::Update()
 				if (m_pBGArm && m_pBGArm->IsAttached() && GetEquippedBGItem() == NULL)
 					m_pBGArm->SetHandPos(m_Pos + m_HolsterOffset.GetXFlipped(m_HFlipped));
 			}
+            // Only reload if no other pickuppable item is in reach
+            if (!pDevice->IsFull() && m_Controller.IsState(WEAPON_RELOAD) && !m_pItemInReach)
+            {
+                if (m_pBGArm && m_pBGArm->IsAttached() && GetEquippedBGItem() == NULL) {
```

Comment:
Took me a while to remember, but it's reorganized because of the HolsterOffset hand position being assigned after this part. I put it in the earlier part so that the hand could move to the magazine for a single frame when the reload starts, same as it does when it ends. Call it insignificant, but I think it's a stepping stone to making proper reload animations lol.
![reloading](https://user-images.githubusercontent.com/55289786/114769182-8a79c200-9d72-11eb-8522-9a6a09ac0cdf.gif)
(^totally not how it actually looks btw)